Makefile

```diff
@@ -591,6 +592,23 @@ copywrite: ## Download copywrite locally if necessary.
 yq: ## Download yq locally if necessary.
 	@./hack/install_yq.sh
 
+.PHONY: operator-sdk
```

Comment:
Looks good! I wonder if we could maybe update it to like the `yq` make target. That will ensure that we have expected version of the sdk, as well as guard against any GH rate-limiters. Might be worth doing now.

Reply:
Sure, changed in 4ceb1e2

Reply:
Thanks!